examples/solid-spa-tailwind-shoelace/renderer/types.ts

```diff
@@ -1,12 +1,9 @@
+import type { PageContextBuiltInClient } from "vite-plugin-ssr/client";
 import type { PageContextBuiltIn } from "vite-plugin-ssr";
-import { Component } from "solid-js";
-
-/**
- * PageContext used by the renderer.
- *
- * vite-plugin-ssr is framework agnostic. Hence, `Page` is defined as `any`.
```

Comment:
Interesting. Why did you take out that comment? 

Using `Omit` is difficult to understand without this comment, don't you think?

Reply:
I agree, see https://github.com/brillout/vite-plugin-ssr/commit/4e3ac790f165c73c6189630cef08b04129f4905a and https://github.com/brillout/vite-plugin-ssr/commit/ce0db176f99864e99697198ef1ecaf6da7308c2d.

Reply:
It might be better to define Page as `unknown` instead of `any` to increase type safety, see https://devblogs.microsoft.com/typescript/announcing-typescript-3-0-rc-2/#the-unknown-type. 

However, the docs should then probably clarify that Page is to be set by the developer.

Reply:
I agree from a type safety POV.

But, from a DX POV, I prefer `pageContext.Page: any`: I don't want to force users to have to define `Page` while `Page` type safety isn't really worth it.